### ISSUE TITLE
Accuracy of  0.5619

### ISSUE THREAD

**Issue by RXAldreezee** [aqibsaeed/Urban-Sound-Classification#38]
Hi, I'm a student studying RNN. I have changed the number of epoch from 3 to 50 of the RNN. After training the accuracy is about 0.5619018899057902. Is this correct?

**Comment by Nisarg-1406** [comment]
You are unnecessary increasing the number of epochs. Due to this, you are overfitting the model and hence, the model is not learning the data but it is memorizing the data which is not required.

**Comment by RXAldreezee** [comment]
Ok, I was just about learning about RNN, I will take note of this. Is there anything to do to increase the accuracy?

**Comment by aqibsaeed** [comment]
Read [Chapter 7 Regularization for Deep Learning, Deep Learning, 2016.](https://www.deeplearningbook.org/contents/regularization.html) to understand what can be done to avoid overfitting.
